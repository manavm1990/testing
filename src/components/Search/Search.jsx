import PropTypes from "prop-types";
import React from "react";

function Search({ changeHandler }) {
  return (
    <div>
      <label htmlFor="search" className="sr-only">
        Search 🔍
      </label>
      <input
        id="search"
        placeholder="Search 🔍"
        type="search"
        onChange={(e) => {
          changeHandler(e.target.value);
        }}
      />
    </div>
  );
}

Search.propTypes = {
  changeHandler: PropTypes.func.isRequired,
};

export default Search;
