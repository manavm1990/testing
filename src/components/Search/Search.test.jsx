import React from "react";
import renderer from "react-test-renderer";
import Search from "./Search";

describe("My Test Suite", () => {
  it("My Test Case", () => {
    expect(true).toBe(true);
  });
});

describe("Search", () => {
  test("snapshot renders", () => {
    const component = renderer.create(<Search />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
