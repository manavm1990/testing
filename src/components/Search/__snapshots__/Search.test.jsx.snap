// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search snapshot renders 1`] = `
<div>
  <label
    className="sr-only"
    htmlFor="search"
  >
    Search 🔍
  </label>
  <input
    id="search"
    onChange={[Function]}
    placeholder="Search 🔍"
    type="search"
  />
</div>
`;
